### PR TITLE
Bug Bash: Runs pt 1

### PR DESCRIPTION
**Pull request by luca3rd**
# What does this PR do?

Updates the docs around the Run CLI experience.

### CODE REVIEW

File: docs/features/runs/2_private-repos.md
Comment (oojo12) [inline docs/features/runs/2_private-repos.md:20]: Reminder to check this before merging and readding if I deleted that content by accident.

File: docs/features/runs/1_README.md
Comment (oojo12) [inline docs/features/runs/1_README.md:93]: Reminder to have this reviewed by Adam prior to merging.

File: docs/features/runs/1_README.md
Comment (oojo12) [inline docs/features/runs/1_README.md:36]: Reminder that these videos should be updated and this comment removed.

File: docs/features/runs/1_Creating Runs/2_Adv Runs/3_creating-runs-from-config.md
Comment (oojo12) [inline docs/features/runs/1_Creating Runs/2_Adv Runs/3_creating-runs-from-config.md:48]: Reminder to remove this prior to merging. @luca3rd have you created a ticket for this? If not I can just make a more complicated config example on https://github.com/PyTorchLightning/grid-tutorials that potentially can be used?

**Review by oojo12 — APPROVED** [review]

**Comment by oojo12** [comment]
@essiequoi do not merge this prior to those reminders being resolved.

**Comment by oojo12** [comment]
Also thank you for the hard work and fixes/additions here @luca3rd appreciate it.

File: docs/cli.md
Comment (essiequoi) [inline docs/cli.md:898]: @pritamsoni-hsr these are all really good changes that will be over-written, so we need to make these changes in the template that generate the CLI docs. 
cc @oojo12

**Review by essiequoi — COMMENTED** [review]
@oojo12 docs/features/runs/1_Creating Runs/1_Basic Runs/1_README.md is deleted in this PR. Is that intended?

**Comment by oojo12** [comment]
> @oojo12 docs/features/runs/1_Creating Runs/1_Basic Runs/1_README.md is deleted in this PR. Is that intended?
@essiequoi 
It was "fill me" content. Ideally, something similar to what is done for creating advanced runs or the runs overview would have replaced that content. But it may be overkill so I am overlooking it.

**Comment by oojo12** [comment]
Slipping in a note on how to use Private Pypi Repos with Grid Run @essiequoi. @luca3rd is okay with this we chatted offline

File: docs/features/runs/1_Creating Runs/2_Adv Runs/3_creating-runs-from-config.md
Comment (oojo12) [reply docs/features/runs/1_Creating Runs/2_Adv Runs/3_creating-runs-from-config.md:48]: Actually this aside does not appear on local build. We can disregard it.

**Comment by adam-grid** [comment]
I think overall it's ok.
Important to remember that cli.md file get's generated from the cli commands descriptions - so the changes have to be made there to be reflected in docs here. -- But I'm not sure (haven't tested) how to make sure the new-line symbols in cli and here in md docs get interpreted correctly in both places. We see the issue here that the table get's malformed if there's the \b symbol in cli docs.

**Comment by adam-grid** [comment]
Another note would be about the cli and script parameters, the usage of short-flags (e.g. -d) or no-flag arguments are really unstable if used with grid_search / random_search sweep strategies. I would keep strong encouragement to use `--flags` if possible

**Comment by oojo12** [comment]
Tried to create the video however the Run fails and is blocked by resolution of this issue https://linear.app/gridai/issue/GRID-8743/grid-fails-to-build-when-reqs-arent-at-root-or-specified.

**Review by essiequoi — APPROVED** [review]

**Pull request merged**
